# HG changeset patch
# User Thorsten Meyer <thorsten.meyier@gmx.de>
# Date 1297892051 -3600
#      Wed Feb 16 22:34:11 2011 +0100
# Node ID 9337a5c234da1e32582bcc34e5b7448ada397892
# Parent  885f524d3771170b33e0c63d1e0ac2b514be91d5
Fix tests in mean.m according to changeset 0f21f258aa17.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-02-16  Thorsten Meyer  <thorsten.meyier@gmx.de>
+
+	* statistics/base/mean.m: Fix tests according to changeset 0f21f258aa17.
+
 2011-02-15  Konstantinos Poulios  <logari81@gmail.com>
 
 	* plot/__fltk_print__.m: Forward pipeline to drawnow instead of
 	invoking a process here.  Bug #32319.
 
 2010-02-14  Rik  <octave@nomad.inbox5.com>
 
 	* plot/semilogxerr.m, plot/semilogyerr.m, special-matrix/pascal.m,
diff --git a/scripts/statistics/base/mean.m b/scripts/statistics/base/mean.m
--- a/scripts/statistics/base/mean.m
+++ b/scripts/statistics/base/mean.m
@@ -134,21 +134,21 @@ endfunction
 %! y = x';
 %! z = [y, y+10];
 %! assert(mean (x) == 0);
 %! assert(mean (y) == 0);
 %! assert(mean (z) == [0, 10]);
 
 %!assert(mean ([2 8], 'g'), 4);
 %!assert(mean ([4 4 2], 'h'), 3);
+%!assert(mean (logical ([1 0 1 1])), 0.75);
 
 %% Test input validation
 %!error mean ();
 %!error mean (1, 2, 3, 4);
 %!error mean ({1:5});
-%!error mean (true(1, 5));
 %!error mean (1, 2, 3);
 %!error mean (1, ones(2,2));
 %!error mean (1, 1.5);
 %!error mean (1, 0);
 %!error mean (1, 3);
 %!error mean (1, 'b');
 
